services/tar/internal/cmd/meta.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	def "go.beyondstorage.io/v5/definitions"
+	"go.beyondstorage.io/v5/types"
+)
+
+var Metadata = def.Metadata{
+	Name:  "tar",
+	Pairs: []def.Pair{},
+	Infos: []def.Info{},
+	Factory: []def.Pair{
+		def.PairEndpoint,
+		def.PairWorkDir,
+	},
+	Service: def.Service{
+		Features: types.ServiceFeatures{},
+	},
+	Storage: def.Storage{
+		Features: types.StorageFeatures{
+			WriteEmptyObject: true,
+
+			Delete:   true,
+			List:     true,
+			Metadata: true,
+			Read:     true,
+			Stat:     true,
+			Write:    true,
```

Comment:
In fact, only `List`, `Read` and `Stat` are currently implemented.

Reply:
Got it, I will check it.